loadtest/report/output.go

```diff
@@ -48,16 +48,26 @@ func printSummary(c comp, target *os.File, base Report, cols int) {
 			} else if metric == "p99" {
 				d = measurement[1]
 			}
+			// We only handle comparisons between 2 reports.
 			if len(d) != 1 {
 				break
 			}
-			if (showImproved && d[0].deltaPercent >= 0) || (!showImproved && d[0].deltaPercent <= 0) {
+			// Skip delta percentages smaller than 1.
+			if math.Abs(d[0].deltaPercent) < 1 {
 				break
 			}
-			fmt.Fprintf(target, "| %s", label)
-			fmt.Fprintf(target, " |  %s", metric)
-			fmt.Fprintf(target, "| %s", getDuration(float64(base.AvgStoreTimes[label])))
-			fmt.Fprintf(target, "| %s | %s | %.3f", d[0].actual, d[0].delta, d[0].deltaPercent)
+			// Only show requested data.
+			if showImproved && d[0].delta > 0 || !showImproved && d[0].delta < 0 {
+				break
+			}
+			// Skip deltas smaller than 2ms.
```

Comment:
Should we give an info about this to the end of the output? something like "deltas below 2ms are discarded".

Reply:
We could document this at https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/compare.md.